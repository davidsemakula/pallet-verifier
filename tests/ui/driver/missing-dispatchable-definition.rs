use std::marker::PhantomData;

pub trait Config {}

pub enum Call<T: Config> {
    __Ignore(PhantomData<T>),
    do_something { val: u8 },
}

pub struct Pallet<T: Config>(PhantomData<T>);
