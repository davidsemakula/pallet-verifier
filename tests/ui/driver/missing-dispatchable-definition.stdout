Searching for FRAME pallet definition ...
Searching for dispatchable function definitions ...
Searching for public associated function definitions ...
